doc/manuals/admin/logs.md

```diff
@@ -108,52 +110,76 @@ The different fields in each line are as follows:
     include the name of the file and line number generating the trace
     (this information is useful mainly for Orion developers).
 
-[Top](#top)
-    
-## Error and warning types
+[Top](#top)  
+
+## Alarms
+
+Alarm conditions:
```

Comment:
This table is very hard to see in the diff view. I'd recomend to have a look to it in the "View" buttom.